src/components/UserDashboard/MySavedQueries.js

```diff
@@ -114,47 +112,52 @@ const MySavedQueries = compose(
             </Span>
           </Flex>
         </Header>
-        <Box overflowY="auto" mt={2} mb={2}>
-          {queries
-            .filter(q => q.alias)
-            .map(q => ({
-              ...q,
-              date: +new Date(q.creationDate),
-              // TODO: save origin + pathname separately in dynamo
-              link: `/search${q.content.longUrl.split('/search')[1]}`,
-            }))
-            .slice()
-            .sort((a, b) => b.date - a.date)
-            .map(q => (
-              <Query key={q.id} inactive={deletingIds.includes(q.id)}>
-                <Column width="100%">
-                  <Row justifyContent="space-between" width="100%">
-                    <Link fontSize={1} fontWeight="bold" color={theme.primary} to={q.link}>
-                      {q.alias}
-                    </Link>
-                    <Box pr={2} pl={2}>
-                      <Span
-                        color={theme.primary}
-                        hover={{ cursor: 'pointer', color: theme.hover }}
-                        onClick={() => deleteQuery({ api, queryId: q.id })}
-                      >
-                        <TrashIcon />
-                      </Span>
-                    </Box>
-                  </Row>
-                  <Box mt={2} mb={2} color={theme.greyScale9} fontSize="0.75rem">
-                    <Detail>{(q.content.Files || 0).toLocaleString()}</Detail> Files |{' '}
-                    <Detail>{(q.content.Participants || 0).toLocaleString()}</Detail> Participants |{' '}
-                    <Detail>{(q.content.Families || 0).toLocaleString()}</Detail> Families |{' '}
-                    <Detail>{q.content.Size}</Detail>
-                  </Box>
-                  <Box fontSize="0.75rem">
-                    Saved {distanceInWords(new Date(), new Date(q.creationDate))} ago
-                  </Box>
-                </Column>
-              </Query>
-            ))}
-        </Box>
+        {queries.length <= 0 ? (
+          <PromptMessageContainer info my={20}>
+            <PromptMessageHeading info mb={10}>
+              You have no saved queries yet.
+            </PromptMessageHeading>
+            <PromptMessageContent>
+              Explore the{' '}
+              <Link to="/search/file" style={{ textDecoration: 'none' }}>
+                File Repository
+              </Link>{' '}
+              and start saving queries!
+            </PromptMessageContent>
+          </PromptMessageContainer>
+        ) : null}
+        <Fragment>
+          <Box mt={2} mb={2}>
+            {queries
+              .filter(q => q.alias)
+              .map(q => ({
+                ...q,
+                date: +new Date(q.creationDate),
```

Comment:
I find the `+` trippy, an unfamiliar eye might conclude it's a typo. Maybe `Number(new Date(q.creationDate))`?